modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/main/resources/META-INF/resources/css/main.scss

```diff
@@ -640,6 +637,22 @@ p.ddm-label {
 	word-wrap: break-word;
 }
 
+.react-field-base-flex-container {
+	display: flex;
+}
+
+.react-field-base-flex-element {
+	flex-grow: 1;
+}
+
+.react-field-base-label {
+	word-break: break-word;
+}
+
+.react-field-base-legend {
+	word-break: break-word;
+}
+
```

Comment:
@mateusgomes01, I don't think that is necessary to use the prefix `react` on these CSS classes, because the prefix `react` was added just because we had another field base component written in MetalJS, so the prefix helped us distinguish both components. In addition, considering that we'll remove the MetalJS component soon, then let's remove the `react` prefix for now, is this ok for you?

One more thing, on the next pulls, try to add the commits which create the CSS classes before the ones which use the CSS classes, it makes your pull easier to comprehend. :wink: Another alternative is add the CSS class declaration in the same pull in which it's being used.

Reply:
Yes, I agree. Gonna remove the `react` and next time I'll add the CSS classes commit before their uses, or together ;)

Thanks for your feedback!